explainer-server/public/javascripts/explain-editor-plain-js.js

```diff
@@ -43,13 +43,26 @@ function updateCheckboxState() {
  * Tag Search
  */
 
-function processCapiSearchResponse(response){
+
+function processCapiSearchResponseTags(divIdentifier,response,userInterfaceTagDescriptionKey){
+    response.results.forEach(function(tag){
```

Comment:
Maybe it would be easier to write this function if you used this scalajs request library? https://github.com/hmil/RosHTTP